frontend/src/main/scala/bloop/engine/tasks/Tasks.scala

```diff
@@ -271,7 +270,7 @@ object Tasks {
 
   private type Discovered = Map[Framework, List[TaskDef]]
   private[bloop] def discoverTests(analysis: CompileAnalysis,
-                                   frameworks: Array[Framework]): Discovered = {
+                                   frameworks: Array[Framework]): Map[Framework, List[TaskDef]] = {
```

Comment:
Should you either use `Discovered` here or remove the above type alias? Personally I like `Discovered`.

Reply:
I removed it, but lost the change in the rebase. That's the only place where it's used, and it's confusing now that there's `DiscoveredTests` too, so I'll remove it.